projects/components/src/common/interfaces/action-item.interface.ts

```diff
@@ -37,7 +37,7 @@ export type ActionHandlerType<R, T> = (selectedEntities?: R[], handlerData?: T)
  */
 export interface ActionItem<R, T> {
     /**
-     * The i18n key for the text of the button
+     * The i18n key or a translated string for contents of a action button
```

Comment:
What's the use case for passing in already translated text? Isn't that just putting more work on the user? If there is a use case, should we expand this pattern to our other components (namely the data exporter?)

Reply:
When displaying results, we display a VM name, that is not translatable